internal/component/database_observability/mysql/collector/query_samples_test.go

```diff
@@ -34,14 +34,13 @@ func TestQuerySamples(t *testing.T) {
 		errorLine  string
 	}{
 		{
-			name: "select query without wait event",
+			name: "select query",
```

Comment:
I've removed most of the test cases here because we're not parsing the `digest_text` anymore. There's parsing logic and similar test cases in the `query_tables` collector though.